xcode/Subconscious/Shared/UIKit/BlockEditor/BlockEditorModel.swift

```diff
@@ -42,6 +42,11 @@ extension BlockEditor {
         
         /// Is editor saved?
         private(set) var saveState = SaveState.saved
+
+        /// Who are we? The did that identifies our sphere.
+        var ourSphere: Did?
```

Comment:
Not using this yet, but will later to disable editing for not-our-sphere. My intent is to use the same component for 1P and 3P content when we finish work on the block editor.